app/scripts/components/granules/overview.js

```diff
@@ -161,11 +166,14 @@ GranulesOverview.propTypes = {
   stats: PropTypes.object,
   dispatch: PropTypes.func,
   workflowOptions: PropTypes.array,
-  location: PropTypes.object
+  location: PropTypes.object,
+  config: PropTypes.object
 };
 
+export { GranulesOverview };
```

Comment:
Why the extra export?

Reply:
Currently I'm doing the extra export to avoid having to deal with `connect()(GranulesOverview)`. I can look into doing this without the extra export.

Reply:
I just noticed that I'm doing the exact same thing for the reconciliation report tests. So this is 100% fine